### PR TITLE
Add Bazelisk dep and bump ibazel version

### PR DESCRIPTION
**Pull request by bduffany**
The new `ibazel` dep includes a bugfix for https://github.com/bazelbuild/bazel-watcher/issues/386

The bazelisk dep is just for correctness in terms of dev workflow -- `yarn dev` doesn't work properly if bazelisk is missing.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,10 +53,11 @@
...
     "@types/react-virtualized": "^9.21.9",
     "@types/uuid": "^8.3.0",
-    "chalk": "^2.4.2",
+    "chalk": "^4.0.0",
     "escodegen": "^1.11.1",
-    "espree": "^3.5.4",
-    "estraverse": "^4.2.0",
+    "espree": "^7.0.0",
+    "estraverse": "^5.1.0",
+    "fbemitter": "^2.1.1",
```
Comment (siggisim) [inline package.json:60]: i did some work recently to remove fbemitter (it has a dependency that has some minor vulnerability)

is there a reason we need this back?

Comment (bduffany) [reply package.json:59]: Nope, this was my mistake when resolving the merge conflict.

File: package.json [hunk 2]
```diff
@@ -3,7 +3,10 @@
   "version": "1.0.0",
   "main": "index.js",
   "license": "MIT",
-  "scripts": {},
+  "scripts": {
+    "clean": "rm -rf bazel-* && rm -rf node_modules",
```
Comment (siggisim) [inline package.json:7]: i'd go with either `bazel clean` or `bazel clean && rm -rf node_modules`

File: package.json [hunk 3]
```diff
@@ -3,7 +3,10 @@
...
+    "dev": "ibazel run server --compilation_mode=fastbuild"
```
Comment (siggisim) [inline package.json:8]: fastbuild is the default i think

**Review by siggisim — APPROVED** [review]
lgtm

**Pull request merged**